src/core/workspace/setup.ts

```diff
@@ -18,19 +18,20 @@ type DetectRoots = () => Promise<WorkspaceRoot[]>
 export async function setupWorkspaceManager({
 	stateManager,
 	detectRoots,
+	isMultiRootEnabled,
```

Comment:
Why do we want to move this as param and then output the value, instead of calling the featureFlag inside the function?

```suggestion
```

Reply:
Fixed